### DISCUSSION TITLE
Monitor OFF and ON

### DISCUSSION THREAD

**Discussion by negoalex** [kangyu-california/PersistentWindows#157] [General]
Hello,

The app is working great. My question is: When the monitor is turned off and the PC turns on before the montior, the app does not detect the monitor and the page will remain on monitor 1 even if im opening the monitor after. Is there any possibility that when I open the monitor to automatically send the image to monitor 2? Now is sending the webpage on the second monitor only if the monitor is already open. If the monitor is off, the image stays on monitor 1.

Thank you!
![image](https://user-images.githubusercontent.com/88164357/167368752-4caf1d39-6214-46aa-8cd6-42de2bf5e082.png)
![image](https://user-images.githubusercontent.com/88164357/167368805-a08aa959-059e-484a-a08d-c73cc436b500.png)

**Comment by kangyu-california** [comment]
I am not sure if the issue is with TeamViewer or not.

PW is supposed to automatically restore window position when monitor is turned on/off since the second time, that is because PW has no knowledge of expected window position when a monitor is on/off for the first time. 
In the latter case, you want to use restore from disk feature assuming save to disk has been executed before.

**Reply by negoalex** [reply]
I will give you a proper example of whats its happening. At my company we use on 1 PC for 2 TV's. On both TV's when the PC is turned on will automaticaly start 2 webpages that goes on each monitor. If the client is not turning on first both TV's before turning the pc on, the webpages are going just on 1 TV because its not detecting the both TV's. If its turning on the TV after, the app will not detect that the TV was open so I have to go with TeamViewer automatically on that PC and put the webpage on the other screen. There is any setting to automatically detect that screen when its open?

**Reply by kangyu-california** [reply]
PW automatically detect monitor on/off/resolution change event (sent by Windows OS), but Windows may fail to detect such event for some old VGA monitor, which might explain your TV problem.

You may want to look into event viewer for event 9990 reported by PW
![image](https://user-images.githubusercontent.com/59128756/167732534-88f764dd-5bcf-4536-842d-a37796a4c0ad.png)
![image](https://user-images.githubusercontent.com/59128756/167732602-6c348d72-ef68-4761-9312-c98138717721.png)

**Comment by negoalex** [comment]
Everything looks ok on this side, all TV's got HDMI ports. The app detect the second monitor when i have my PC on and I turn off the TV's but if I power ON the pc and the TV's are opened in like 15min after i turned on the pc, everything will be on main screen. Thank u for all the answers btw.
